uktrade/data-hub-frontend
# Investment CSV download policy feedback fix

## Description of change

Fixes a Business intelligence filter issue when downloading a spreadsheet from Interactions.

## Test instructions

Go to `/interactions` and apply the Business intelligence filter and then download the results. The number of rows within the spreadsheet should match the number of items in the filtered collection list.

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
